### PR TITLE
Content change in Sign-up button

### PR DESCRIPTION
**Pull request by aditianshu**

### CODE REVIEW

File: src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx [hunk 1]
```diff
@@ -19,9 +19,17 @@ export const GetStarted: React.FunctionComponent = withErrorBoundary(() => (
       </Paragraph>
     </TopSection>
     <BottomSection>
-      <Button size="large" category="secondary">
-        <Link to="/Signup">Sign-Up</Link>
-      </Button>
+      { localStorage.getItem('authenticated')? 
+        <a href = "https://app.slack.com/client/TRN1H1V43/CUCLB9VJL" target = "_blank">
```
Comment (Marvin9) [inline src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx:23]:
```suggestion
        <a href = "https://app.slack.com/client/TRN1H1V43/CUCLB9VJL" target = "_blank" rel="noopener noreferrer">
```

File: src/ignitus-HomePage/ignitus-Introduction/Components/index.tsx [hunk 2]
```diff
@@ -18,9 +18,15 @@ export const Introduction: React.FunctionComponent = withErrorBoundary(() => (
         Ignite your resume and get the best global opportunities and
         Skyrocketing your career
       </I.PrimaryText>
-      <I.Button category="primary" size="large">
-        SIGN UP
-      </I.Button>
+      {localStorage.getItem('authenticated')? 
+        <a href = "https://app.slack.com/client/TRN1H1V43/CUCLB9VJL" target = "_blank">
```
Comment (Marvin9) [inline src/ignitus-HomePage/ignitus-Introduction/Components/index.tsx:22]: Add rel here too.

File: src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx [hunk 3]
```diff
@@ -19,9 +19,17 @@ export const GetStarted: React.FunctionComponent = withErrorBoundary(() => (
...
+          <Button category="primary" size="large">
```
Comment (Paarmita) [inline src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx:24]: Please change the category of a button from `primary` to `secondary` here.

Comment (aditianshu) [reply src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx]: Done

Comment (Paarmita) [reply src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx]: I still don't see the change. 
Screenshot 2020-05-02 at 7 07 48 PM

Comment (aditianshu) [reply src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx]: Sorry, I must've missed it. Updated.

**Comment by aditianshu** [comment]
image

I fixed some pretty issues that came up, but this check is failing in a file which I haven't changed. What should I change here?

**Comment by Marvin9** [comment]
Don't change anything. It is just minor issue in develop branch.

**Comment by divyanshu-rawat** [comment]
@Marvin9 I think snapshots are not updating via `jest --updateSnapshot`, so there is some issue with jest config I guess, like didn't use to happen earlier. Last time I manually changed updated the snapshot. @aditianshu I will review your or in the evening & update the snapshot manually to fix the build.

**Comment by Marvin9** [comment]
@divyanshu-rawat I will look into it.

@aditianshu write ```npm run lint:fix``` in your terminal & ```npm run test```, if you encounter same error press ```u```, then commit & push.

File: src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx [hunk 4]
```diff
@@ -19,9 +19,21 @@ export const GetStarted: React.FunctionComponent = withErrorBoundary(() => (
...
-      <Button size="large" category="secondary">
-        <Link to="/Signup">Sign-Up</Link>
-      </Button>
+      {localStorage.getItem('authenticated') ? (
+        <a
+          href="https://app.slack.com/client/TRN1H1V43/CUCLB9VJL"
+          target="_blank" 
+          rel="noopener noreferrer"
+        >
+          <Button category="secondary" size="large">
```
Comment (divyanshu-rawat) [inline src/ignitus-HomePage/ignitus-GetStarted/Components/index.tsx:28]: Please change the category of a button to `primary` as seen here http://www.ignitus.org/

**Comment by divyanshu-rawat** [comment]
@Marvin9 snapshots are updating now. 🍻

**Comment by aditianshu** [comment]
Thank you! :)

**Pull request merged**